real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/localizers/HighTrustGPSLocalizer.java

```diff
@@ -39,51 +39,66 @@ public HighTrustGPSLocalizer(){
     	buggyFrameGpsX = 0.0;
     	buggyFrameGpsY = 0.0;
         buggyFrameRotZ = 0.0;
+        buggySteeringAngle = 0.0;// wheel direction in buggy frame
```

Comment:
How about positive steers right, negative steers left? I don't remember what high level or low level does for this but I agree that we need to standardize it